app-backend/db/src/test/scala/com/azavea/rf/database/TaskDaoSpec.scala

```diff
@@ -1505,4 +1505,187 @@ class TaskDaoSpec
       }
     }
   }
+
+  test("updating reviews of tasks should update review status of their parent") {
+    check {
+      forAll {
+        (
+            userCreate: User.Create,
+            orgCreate: Organization.Create,
+            platform: Platform,
+            projectCreate: Project.Create,
+            taskFeaturesCreate: Task.TaskFeatureCollectionCreate,
+            annotationProjectCreate: AnnotationProject.Create
+        ) =>
+          {
+            val connIO =
+              for {
+                (dbUser, _, _, dbProject) <- insertUserOrgPlatProject(
+                  userCreate,
+                  orgCreate,
+                  platform,
+                  projectCreate
+                )
+                dbAnnotationProj <- AnnotationProjectDao
+                  .insert(
+                    annotationProjectCreate.copy(
+                      projectId = Some(dbProject.id)
+                    ),
+                    dbUser
+                  )
+                tasks <- TaskDao.insertTasks(
+                  fixupTaskFeaturesCollection(
+                    taskFeaturesCreate,
+                    dbAnnotationProj
+                  ),
+                  dbUser
+                )
+                parent = tasks.features.take(1)
```

Comment:
is `take` here to accommodate cases in which we get an empty list from the generators? It creates a bit of awkward context below, where because it's lists it's not obvious that `childOneFC` e.g. is 0 or 1 children. I think `headOption` should be a drop in replacement for `.take(1)` that will make the types below more precise about what's going on:

```suggestion
                parent = tasks.features.headOption
```